test/Model.js

```diff
@@ -8,6 +8,12 @@ function config(additions) {
 }
 
 describe("Model", () => {
+  it("does not support number parameters for Allowlist", () => {
+    expect(() => new Model({ allowlist: new Allowlist([123]) })).to.throw(
+      "Unsupported number parameter - Ref https://github.com/IBM/audit-ci/pull/217"
```

Comment:
If you update the above text, this will also need updating.
```suggestion
      "Unsupported number as allowlist. Perform codemod to update config to use GitHub advisory as identifiers: https://github.com/quinnturner/audit-ci-codemod with `npx @quinnturner/audit-ci-codemod`. See also: https://github.com/IBM/audit-ci/pull/217"
```